### PR TITLE
Adding peered_network_ip_range field to tests

### PR DESCRIPTION
**Pull request by khalkie**
Adding new field `peered_network_ip_range` to Google Cloud Build workerpool tests

If this PR is for Terraform, I acknowledge that I have:

- [ ] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [ ] [Generated Terraform](https://github.com/GoogleCloudPlatform/magic-modules#generating-the-terraform-providers), and ran `make test` and `make lint` to ensure it passes unit and linter tests.
- [ ] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/third_party/terraform/tests) (for handwritten resources or update tests).
- [ ] [Ran](https://github.com/hashicorp/terraform-provider-google/blob/main/.github/CONTRIBUTING.md#tests) relevant acceptance tests (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [ ] Read the [Release Notes Guide](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/.ci/RELEASE_NOTES_GUIDE.md) before writing my release note below.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:REPLACEME

```

### CODE REVIEW

File: mmv1/third_party/terraform/tests/resource_cloudbuild_worker_pool_test.go.erb [hunk 1]
```diff
@@ -151,6 +151,7 @@ resource "google_cloudbuild_worker_pool" "pool" {
 	}
 	network_config {
 		peered_network = google_compute_network.network.id
+		peered_network_ip_range = "/25"
```
Comment (rileykarson) [inline mmv1/third_party/terraform/tests/resource_cloudbuild_worker_pool_test.go.erb:154]: This field isn't actually added to the resource yet!

Comment (khalkie) [reply mmv1/third_party/terraform/tests/resource_cloudbuild_worker_pool_test.go.erb:154]: Hi Riley,
Thanks for looking at my CL - I ran the 'make upgrade-dcl' script to add my DCL changes, which fixed this test. However, my changes were not related to the other tests which are failing - how should I proceed?

Comment (rileykarson) [reply mmv1/third_party/terraform/tests/resource_cloudbuild_worker_pool_test.go.erb:154]: The remaining failure- TestAccCloudfunctions2function_cloudfunctions2BasicGcsExample- is failing for an unrelated reason.

**Review by rileykarson — COMMENTED** [review]
Ah, this was an early DCL-based resource and uses a handwritten documentation page. Can you update https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/website/docs/r/cloudbuild_worker_pool.html.markdown to include the new field as well?

**Comment by khalkie** [comment]
Somehow I discarded my commits by attempting to sync my fork, so I recreated this CL here

https://github.com/GoogleCloudPlatform/magic-modules/pull/7335

**Pull request closed without merging**